### ISSUE TITLE
createProvider is not a function

### ISSUE THREAD

**Issue by tofoli** [lolatravel/realm-react-redux#23]
Occurs with the update of react-redux to 6.0.1

```
  React Native Environment Info:
    System:
      OS: macOS 10.14.3
      CPU: (4) x64 Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz
      Memory: 156.46 MB / 8.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.15.0 - ~/.nvm/versions/node/v10.15.0/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.4.1 - ~/.nvm/versions/node/v10.15.0/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
      Android SDK:
        API Levels: 23, 25, 26, 27, 28
        Build Tools: 27.0.3, 28.0.3
    IDEs:
      Xcode: 10.2/10E125 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.6 => 16.8.6 
      react-native: 0.59.3 => 0.59.3 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
```

<img width="298" src="https://user-images.githubusercontent.com/2599384/55510505-d1d20180-5634-11e9-935a-aa1e275d7f66.png">

<img width="420" src="https://user-images.githubusercontent.com/2599384/55510562-eca47600-5634-11e9-8e2c-302a026c1f33.png">

**Comment by lamflam** [comment]
Thanks for reporting @tofoli, I have not tried updating to `react-redux@6` yet but I will try looking into this this week.

**Comment by enahum** [comment]
Anything on this? There is a version 7 now

**Comment by lamflam** [comment]
@enahum I apologize that I have not been able to get to this yet. The main issue was the API change in react-redux 5 -> 6 where they moved to the new context API in React. It _shouldn't_ be terribly hard to update but I haven't had time to investigate. From what I can tell, the `selector` api hasn't changed so none of the core logic should need to be updated.

I'd be happy to merge a PR and create a new release if anyone has the time to make the changes.

**Comment by enahum** [comment]
Hey @lamflam thanks, I do have an updated fork and will be submitting a few PR’s to update it and have a bit more control over the snapshots and stuff.

**Comment by lamflam** [comment]
Awesome! I look forward to the PRs, and I should be able to get them merged and released relatively quickly.

**Comment by lamflam** [comment]
Fixed by #24, released in [v0.0.9](https://www.npmjs.com/package/realm-react-redux).
